UKSI/import-uksi-helper.php

```diff
@@ -87,6 +87,13 @@ public static function getSettings(array $argv, $argc) {
     $settings = array_merge([
       'data-path' => dirname(__FILE__),
     ], $settings);
+    $settings = array_merge([
```

Comment:
* You can combine this array_merge with the one above so all the default settings are done together.
* Also, perhaps this setting should be hyphenated for consistency with data-path?
* Comments should end with . (Drupal standards)

Reply:
Will do. I originally had the name hyphenated but I didn't like the way VS Code styled it so I used underscores!! (Not a good reason I know.)  Thanks for heads-up on the commented standards.